test_util/EclRegressionTest.hpp

```diff
@@ -79,7 +79,7 @@ class ECLRegressionTest: public ECLFilesComparator {
         this->printKeywordOnly = printArg;
     }
 
-    void compareSpesificKeyword(std::string keyword) {
+    void compareSpesificKeyword(const std::string& keyword) {
         this->spesificKeyword = keyword;
```

Comment:
Does the static analyser accept the other way to write this, namely using `std::move`?
```C++
void compareSpesificKeyword(std::string keyword)
{
    this->spesificKeyword = std::move(kw);
}
```

Reply:
yes, rewritten to use move.